test/test-network/systemd-networkd-tests.py

```diff
@@ -158,6 +158,31 @@ def f(func):
 
     return f
 
+def expectedFailureIfNetdevsimWithSRIOVIsNotAvailable():
+    def f(func):
+        call('rmmod netdevsim', stderr=subprocess.STDOUT)
+        rc = call('modprobe netdevsim', stderr=subprocess.STDOUT)
+        if rc != 0:
+            return unittest.expectedFailure(func)
```

Comment:
yes fedora does not support till now.
```
~ sus@esx-fedora
❯ sudo modprobe netdevsim
modprobe: FATAL: Module netdevsim not found in directory /lib/modules/5.8.0-0.rc1.20200617git69119673bd50.1.fc33.x86_64
```

Reply:
kernel-modules-internal-5.6.19-300.fc32.x86_64 provides netdevsim.